cindy/index.js

```diff
@@ -0,0 +1,33 @@
+// 47-PB-datastructure-destructuring
+// Destructuring
+
+// Instructions
+// 1. Array Destructuring
+
+//     Declare the variables fruit, vegetable, food. Assign the following values to the variables respectively: "banana", "cucumber", "bread", "cakes", "pizza" (assign the "bread", "cakes" and "pizza" to food). Use array destructuring assignment to complete this task.
+let items = ["banana", "cucumber", "bread", "cakes", "pizza"]
+const [fruit, , , , ] = items 
+const [, vegetable, , , ] = items
+const [, , ...food] = items
+console.log(`fruit: ${fruit}, vegetable: ${vegetable}, food: ${food}`)
+
+// 2.Object Destructuring
+//     Create an object which contains key pair values of names and the respective halloween costume e.g. fran: "witch". Assign each key pair value to corresponding variable. Print out the object values.
+const halloweenCostume = ['witch','vampire','mummy', ['barbie', 'ken']]
+const [fran,jimmy,jean,kimmy ] = halloweenCostume
+console.log(`fran: ${fran}, jimmy: ${jimmy}, jean: ${jean}, kimmy: ${kimmy}`)
+
+// 3. Parameters: Object Destructuring
+//     Create an object with band/musician details e.g. name of the band/musician, greatest hit, nationality, genre etc.
+//     Then, create a function to display all of the band/musician's data.
+//     Example
+// `Regina Spektor is a American-Russian singer/songwriter and pianist. The musician sings indie-pop and their greatest hit is "Us"`.
+const bandInfo = {
+    name: 'Yeah Yeah Yeahs',
+    nationality: 'The US',
+    genre: 'rock',
+    type: 'band',
+    albums: ['Fever to Hell', 'it`s blizz!', 'Show your Bones','Mosquito','MAS'],
+    members: ['Karen O', 'Nick Zinner', 'Brian Chase'],
+} 
+console.log(`${bandInfo.name} is a ${bandInfo.genre} ${bandInfo.type} from ${bandInfo.nationality}. They have 3 members: ${bandInfo.members} and several Albums like ${bandInfo.albums}`)
```

Comment:
The last Task ==> Please create a function to display all of the band/musician's data instead of just console.log()

Reply:
add missing function! :zap: